.gitignore

```diff
@@ -9,3 +9,6 @@ npm-debug.log
 .DS_Store
 .AppleDouble
 .LSOverride
+
+# Netlify
+.netlify
```

Comment:
Not really familiar with Netifly, but is this a file or folder that gets ignored?

Reply:
Yeah,

Reply:
Sorry, I think part of your response got cut off

Reply:
Oh, I guess my question might not have bee clear. Is this a file or a folder? If it's a folder it should have a trailing slash

Reply:
It's a folder but due a bug in their client, it just adds `.netlify` in .gitignore even if one has `/.netlify/`